fetch_gxa_metadata.py

```diff
@@ -3,63 +3,70 @@
 import subprocess
 import json
 import requests
+import yaml
+import pandas as pd
+import argparse
+import sys
 
-max_retries = 5
-retry_delay = 5
-studyid_counter = 0
-formatted_datetime = datetime.now().strftime("%Y-%m-%d %H:%M:%S")
-gxa_url = "https://www.ebi.ac.uk/gxa/json/experiments/"
+MAX_RETRIES = 5
+RETRY_DELAY = 5
+GXA_URL = "https://www.ebi.ac.uk/gxa/json/experiments/"
+
+if len(sys.argv) != 3:
+    print("Usage: python fetch_gxa_metadata.py <output_yaml_filename> <output_tsv_filename>")
+    sys.exit(1)
+
+parser = argparse.ArgumentParser(description="Generate and save experiment data to YAML and tsv")
+parser.add_argument("yaml_filename", help="Output Yaml filename")
+parser.add_argument("tsv_filename", help="Output Tsv filename")
```

Comment:
Are these both `yaml_filename` and `yaml_filename` required or either is fine.. 

if aim is to generate both file per run, consider taking just one optional argument like `file_name` with some default value and just change extension so user need to provide only one argument or none.

if aim is to generate either then script needs some logic at the EOF

Reply:
[e7a4e05](https://github.com/ebi-gene-expression-group/atlas-experiment-fetcher/pull/1/commits/e7a4e051312ccb4e9a0da0cf7b2a5257ff5813d6)